.changeset/smart-yaks-walk.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sumup/circuit-ui': minor
+---
+
+Added a z-index to the Toast ListWrapper.
```

Comment:
The ListWrapper is an implementation detail that users don't know and shouldn't care about. 

```suggestion
Added a `z-index` to the NotificationToasts so they always float above other content.
```